scripts/g.bands/testsuite/test_g_bands.py

```diff
@@ -18,18 +19,18 @@ def test_number_system_defined(self):
 
         # get number of valid band identifiers by Bands lib
         from grass.bandref import BandReferenceReader
+
         nbands_ref = len(BandReferenceReader().get_bands())
 
         self.assertEqual(nbands, nbands_ref)
 
     def test_number_s2(self):
         # test number of S2 band identifiers (hardcoded, no changes expected)
         #
-        nbands = self._number_of_bands(pattern='S2')
+        nbands = self._number_of_bands(pattern="S2")
 
         self.assertEqual(nbands, 13)
 
-
     def test_number_s2_1(self):
         # test if S2_1 is defined (lower + upper case)
```

Comment:
pls fix `'S2_1'` --> `"S2_1"` two lines below which I broke in the merge conflict resolution (I cannot reach the line here to edit it myself)

Reply:
You might be looking for _Jump to... > {filename} > ⋯ > Edit file_.